src/include/stir/Shape/GenerateImage.h

```diff
@@ -0,0 +1,98 @@
+//
+//
+/*
+    Copyright (C) 2022, University College London
+    This file is part of STIR.
+
+    SPDX-License-Identifier: Apache-2.0
+
+    See STIR/LICENSE.txt for details
+*/
+/*!
+  \file
+  \ingroup Shape
+
+  \brief Declaration of class stir::GenerateImage
+
+  \author Kris Thielemans
+  \author Sanida Mustafovic
+  \author Robert Twyman
+
+  *See generate_image utility documentation for parameter file example and usage documentation.
```

Comment:
Documentation of the parameter files here? We don't particulaly want duplicate documentation?

Reply:
As this class can be exposed to Python, it indeed makes most sense to move doc here, and refer to it in the utility